pkg/controller/cache/redis.go

```diff
@@ -18,297 +18,159 @@ package cache
 
 import (
 	"context"
-	"time"
+	"fmt"
+	"strconv"
+	"strings"
 
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/redis/mgmt/redis/redisapi"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	runtimev1alpha1 "github.com/crossplaneio/crossplane-runtime/apis/core/v1alpha1"
-	"github.com/crossplaneio/crossplane-runtime/pkg/logging"
 	"github.com/crossplaneio/crossplane-runtime/pkg/meta"
 	"github.com/crossplaneio/crossplane-runtime/pkg/resource"
 
 	"github.com/crossplaneio/stack-azure/apis/cache/v1alpha3"
 	azurev1alpha3 "github.com/crossplaneio/stack-azure/apis/v1alpha3"
-	azure "github.com/crossplaneio/stack-azure/pkg/clients"
+	common "github.com/crossplaneio/stack-azure/pkg/clients"
```

Comment:
What is the thinking behind this import alias? I don't recall any other v1beta1 controllers using this naming convention for their clients package.

Reply:
Probably, it was probably either too early in the morning or too late in the night when I unconsciously decided to change that. Corrected.